### PR TITLE
2019 03 notes

### PR DESCRIPTION
**Pull request by ChristinaLK**
@karenword @raynamharris 

I cleaned up the notes a little bit, so it's not a 100% transcription, but should be close.

### CODE REVIEW

File: minutes/2019-03-07-trainers.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+Trainer meeting, March 7, 2019 UTC 14:00 
+
+See this link for your local time: See this link for your local time: https://www.timeanddate.com/worldclock/fixedtime.html?iso=20190307T14
+
+**Training discussion signup (pre and post)**
+
+* Juliane Schneider (post)
+* Sarah Stevens (post)
+* Gerg Wislon (post) - what Sarah said...
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: Greg Wilson

File: minutes/2019-03-07-trainers.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+* Ana Costa Conrado (late, sorry)
+
+### Training Discussion
+
+Tip: general impression, highs and lows, anything useful for others to know
+
+Juliane and Serah's workshop. 
+* Serah's first time. 
+* Some city-wide internet difficulties due to  weather. Used zoom without video. Etherpad functionality lost on day 2, but it was very useful and enjoyable when it was working. 
+* Teaching exercising made a noticable impact on student engagement.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: noticable  -> noticeable

File: minutes/2019-03-07-trainers.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+Juliane and Serah's workshop. 
+* Serah's first time. 
+* Some city-wide internet difficulties due to  weather. Used zoom without video. Etherpad functionality lost on day 2, but it was very useful and enjoyable when it was working. 
+* Teaching exercising made a noticable impact on student engagement.  
+
+Juliane and Chris? previous workshop, have over 60% checkout completion. 
+
+Sarah Stevens and GW 
+* Sarah's first time teaching online. 
+	* High: sharing shared exeriences.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: experiences

File: minutes/2019-03-07-trainers.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+Sarah Stevens and GW 
+* Sarah's first time teaching online. 
+	* High: sharing shared exeriences.  
+	* Low: some rough edges due to first time. 
+* New goal: work on inclusive langage by minimzing use of "guys" rather than something like "folks". 
+* GW says Sarah was excellent. 
+* Noticed a lack of continutiy; suggested smoothing of the lesson. 
+* Priority for this year is replacing Etherpad, which is a software that is no longer supported and is lacking functionality for concept maps. 
+* Lots of participation and ethusiams from group and solo participants.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: enthusiasm

File: minutes/2019-03-07-trainers.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+Juliane and Chris? previous workshop, have over 60% checkout completion. 
+
+Sarah Stevens and GW 
+* Sarah's first time teaching online. 
+	* High: sharing shared exeriences.  
+	* Low: some rough edges due to first time. 
+* New goal: work on inclusive langage by minimzing use of "guys" rather than something like "folks". 
+* GW says Sarah was excellent. 
+* Noticed a lack of continutiy; suggested smoothing of the lesson.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: continuity

File: minutes/2019-03-07-trainers.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+* Some city-wide internet difficulties due to  weather. Used zoom without video. Etherpad functionality lost on day 2, but it was very useful and enjoyable when it was working. 
+* Teaching exercising made a noticable impact on student engagement.  
+
+Juliane and Chris? previous workshop, have over 60% checkout completion. 
+
+Sarah Stevens and GW 
+* Sarah's first time teaching online. 
+	* High: sharing shared exeriences.  
+	* Low: some rough edges due to first time. 
+* New goal: work on inclusive langage by minimzing use of "guys" rather than something like "folks".
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: language by minimizing

File: minutes/2019-03-07-trainers.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+Sarah Stevens and GW 
+* Sarah's first time teaching online. 
+	* High: sharing shared exeriences.  
+	* Low: some rough edges due to first time. 
+* New goal: work on inclusive langage by minimzing use of "guys" rather than something like "folks". 
+* GW says Sarah was excellent. 
+* Noticed a lack of continutiy; suggested smoothing of the lesson. 
+* Priority for this year is replacing Etherpad, which is a software that is no longer supported and is lacking functionality for concept maps. 
+* Lots of participation and ethusiams from group and solo participants. 
+* Comment: a spring push for revising cirriculum is tentatively planned and the goal is to only fix minor issues. fixing continutiy is a good thing.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: curriculum and continuity

File: minutes/2019-03-07-trainers.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+* GW says Sarah was excellent. 
+* Noticed a lack of continutiy; suggested smoothing of the lesson. 
+* Priority for this year is replacing Etherpad, which is a software that is no longer supported and is lacking functionality for concept maps. 
+* Lots of participation and ethusiams from group and solo participants. 
+* Comment: a spring push for revising cirriculum is tentatively planned and the goal is to only fix minor issues. fixing continutiy is a good thing. 
+
+Mark: looking forward to teaching next week. All good to go. 
+
+Greg and Amy: were supposed to teach two 1 day events in Toronto (Carpentry inspired) but weather did not permit. 
+* Confident that a 1 day can have the same effect on teaching practice. Goal should be to anlyze what teaching practices get adopted. Could separate out ____ from _____. Could deliver more training with the same impact and effort.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: analyze

File: minutes/2019-03-07-trainers.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+* Priority for this year is replacing Etherpad, which is a software that is no longer supported and is lacking functionality for concept maps. 
+* Lots of participation and ethusiams from group and solo participants. 
+* Comment: a spring push for revising cirriculum is tentatively planned and the goal is to only fix minor issues. fixing continutiy is a good thing. 
+
+Mark: looking forward to teaching next week. All good to go. 
+
+Greg and Amy: were supposed to teach two 1 day events in Toronto (Carpentry inspired) but weather did not permit. 
+* Confident that a 1 day can have the same effect on teaching practice. Goal should be to anlyze what teaching practices get adopted. Could separate out ____ from _____. Could deliver more training with the same impact and effort. 
+	* Comment: if we explore a 1 day format, should we keep it as 1 day lecture/learn and 1 day do?
+	* Comment: Another reason to explor the 1 day format is because of the divergence of the "mechanics for workshops" is not a one size fits all for SWC, LC, DC. Could customize activites more for day 2.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: explore and activities

File: minutes/2019-03-07-trainers.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+	* Comment: if we explore a 1 day format, should we keep it as 1 day lecture/learn and 1 day do?
+	* Comment: Another reason to explor the 1 day format is because of the divergence of the "mechanics for workshops" is not a one size fits all for SWC, LC, DC. Could customize activites more for day 2. 
+
+### Announcements 
+
+* Help wanted for teaching demos! https://pad.carpentries.org/teaching-demos-recovered
+	* now that we have lots of trainers, we should be able to cover this. at one point there was a quota of 1 per quarter. https://docs.carpentries.org/topic_folders/instructor_training/duties_agreement.html
+* Want to schedule your own teaching demo time slot? We are now piloting the use of Calendly to assist with this. If you'd like to help test out the system (and schedule a demo!) contact Karen krword@carpentries.org
+	* due to some conversations on slack, some of the current time slots do not work for trainers. contact Karen if you would like to set up your own if the time slot doesnt' work for you.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: doesn't

File: minutes/2019-03-07-trainers.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+
+### Announcements 
+
+* Help wanted for teaching demos! https://pad.carpentries.org/teaching-demos-recovered
+	* now that we have lots of trainers, we should be able to cover this. at one point there was a quota of 1 per quarter. https://docs.carpentries.org/topic_folders/instructor_training/duties_agreement.html
+* Want to schedule your own teaching demo time slot? We are now piloting the use of Calendly to assist with this. If you'd like to help test out the system (and schedule a demo!) contact Karen krword@carpentries.org
+	* due to some conversations on slack, some of the current time slots do not work for trainers. contact Karen if you would like to set up your own if the time slot doesnt' work for you. 
+* We're thinking about trainer status (e.g. active, emeritus). Fill out this form to generate more discussion later https://goo.gl/forms/qSqrtRMj6B5Na3gc2
+	* for trainers who want to step back from training for a brief timeperiod or forever, we want to help and accomdate that. we want to explore your ideas for the best way to do this.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: accommodate

File: minutes/2019-03-07-trainers.md [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+* Want to schedule your own teaching demo time slot? We are now piloting the use of Calendly to assist with this. If you'd like to help test out the system (and schedule a demo!) contact Karen krword@carpentries.org
+	* due to some conversations on slack, some of the current time slots do not work for trainers. contact Karen if you would like to set up your own if the time slot doesnt' work for you. 
+* We're thinking about trainer status (e.g. active, emeritus). Fill out this form to generate more discussion later https://goo.gl/forms/qSqrtRMj6B5Na3gc2
+	* for trainers who want to step back from training for a brief timeperiod or forever, we want to help and accomdate that. we want to explore your ideas for the best way to do this. 
+* Would like 1 more reviewer for Open Instructor Training applications, reading free-response questions for subjective evaluation only, 5-10 applications every 2 weeks. Contact Karen krword@carpentries.org
+	* assistance needed. pleas contact Karen if you can assist
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: please

File: minutes/2019-03-07-trainers.md [hunk 13]
```diff
@@ -0,0 +1,207 @@
...
+* In-person trainings: please emphasize group code entry in applications (wrong code is better than no code!) -- this keeps people from being unnecessarily reviewed in our Open training pool.
+	* The way application process works, there is an application leaners fill out. this puts them in our system and allows up to start tracking them. Codes are given to member sites. Open trainess lack codes.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: Open trainings

File: minutes/2019-03-07-trainers.md [hunk 14]
```diff
@@ -0,0 +1,207 @@
...
+	* Sher is happy to assist in helping the organizing and coordinating the logistics needed for in-person workshops 
+* Library Carpentry checkout: now (well, tomorrow) with badging! woohoo!
+	* fine tuning the system to record badges. unrolling this new system tomorrow or over the weekend. 
+* Revisions to the Code of Conduct Documentation including a tl;dr: https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html
+	* please read the changes! lots of effort went into this? 
+* RStudio Instructor Training program has launched: https://blog.rstudio.com/2019/02/28/rstudio-instructor-training/
+	* primarily aimed at people who are offering teaching in their workplace or contract based. Day 1 is pedagogy, Day 2 is aimed at making sure people do not teach out-of-date practices. want to make it so people are confident they are getting high quality teaching. Carpentires trainers will be given a voucher(only need to answer a few technical questions - to ensure they are up to date with the technology) to be certified. Certification will allow you to be paid to teach (corporate training). Great for professional development.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: Carpentries

File: minutes/2019-03-07-trainers.md [hunk 15]
```diff
@@ -0,0 +1,207 @@
...
+- Once upon a time, only Software Carpentry; then Data Carpentry, recently Library Carpentry
+- In the past, then, you got a "SWC" badge to teach (b/c that was it)
+- Once DC was launched and of a certain size, SWC instructors could go through a short orientation (w/ Tracy) and get a DC badge
+- Now, with the addition of Library Carpentry, can be badged that way.
+- Used to have to do two demos to get badged for both programs
+
+Now:
+
+- You get ONE badge, for the demo that you did
+- But you can teach any Carpentries workshops you want or feel confortable with
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: comfortable

File: minutes/2019-03-07-trainers.md [hunk 16]
```diff
@@ -0,0 +1,207 @@
...
+
+Comments and questions. 
+- badge is based on the lesson they used for demo. Staff and other leaderships trust that they will select what they are qualified for. Not everyone keeps their expertise up to date in AMY. 
+- In discussions regarding have a unviersal "Carpentries"badge, some people want to identify strongly with one or the other, however others want a Carpentires badge. .
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: universal and Carpentries

In discussions regarding having a universal "Carpentries" badge, some people want to identify strongly with one or the other; however, others want a single "Carpentries" badge.

File: minutes/2019-03-07-trainers.md [hunk 17]
```diff
@@ -0,0 +1,207 @@
...
+Now:
+
+- You get ONE badge, for the demo that you did
+- But you can teach any Carpentries workshops you want or feel confortable with
+
+Comments and questions. 
+- badge is based on the lesson they used for demo. Staff and other leaderships trust that they will select what they are qualified for. Not everyone keeps their expertise up to date in AMY. 
+- In discussions regarding have a unviersal "Carpentries"badge, some people want to identify strongly with one or the other, however others want a Carpentires badge. . 
+- TTT is more about can you teach not what you teach self selection appears to be working quite well. 
+- Some inconsisteny with badges - a badge might be given if based on the lesson PR if the demo lesson isn't reported.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: inconsistency

File: minutes/2019-03-07-trainers.md [hunk 18]
```diff
@@ -0,0 +1,207 @@
...
+- But you can teach any Carpentries workshops you want or feel confortable with
+
+Comments and questions. 
+- badge is based on the lesson they used for demo. Staff and other leaderships trust that they will select what they are qualified for. Not everyone keeps their expertise up to date in AMY. 
+- In discussions regarding have a unviersal "Carpentries"badge, some people want to identify strongly with one or the other, however others want a Carpentires badge. . 
+- TTT is more about can you teach not what you teach self selection appears to be working quite well. 
+- Some inconsisteny with badges - a badge might be given if based on the lesson PR if the demo lesson isn't reported. 
+- From a new trainer, there was some confusion/wonder regarding which badge and what that meant for ability to teach. 
+- I think the desire for a universal badge will increase more and more when people joining the community post-merger +1
+- A stronger devide in the community is R versus python
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: divide

File: minutes/2019-03-07-trainers.md [hunk 19]
```diff
@@ -0,0 +1,207 @@
...
+- A universal badge would be simpler.
+- the pedagoy/inclusivity/motivation is what unifies us. so a universal badge makes sense. then they could decorate that badge if identiy if important. unity across the domains is inclusive. +1 +1
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: pedagogy

File: minutes/2019-03-07-trainers.md [hunk 20]
```diff
@@ -0,0 +1,207 @@
...
+- TTT is about the pedagody not the lessons. Let people self identify under a general badge. But can also bring in heiractical or historical sentiments that are not helpful.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: pedagogy and hierarchical

File: minutes/2019-03-07-trainers.md [hunk 21]
```diff
@@ -0,0 +1,207 @@
...
+- From a new trainer, there was some confusion/wonder regarding which badge and what that meant for ability to teach. 
+- I think the desire for a universal badge will increase more and more when people joining the community post-merger +1
+- A stronger devide in the community is R versus python
+- A universal badge would be simpler.
+- the pedagoy/inclusivity/motivation is what unifies us. so a universal badge makes sense. then they could decorate that badge if identiy if important. unity across the domains is inclusive. +1 +1
+- TTT is about the pedagody not the lessons. Let people self identify under a general badge. But can also bring in heiractical or historical sentiments that are not helpful. 
+
+End goal is that badging is 1) transparent and clear 2) and useful. 
+
+If we slim training to 1 day where there is more of a separation of pedagoy and lesson-specific mechanism, this could become more important. Developing a community for imrporved discourse. -> Often why badges rates are low because some people are just there for professional development.
```
Comment (raynamharris) [inline minutes/2019-03-07-trainers.md]: pedagogy and improved

File: minutes/2019-03-07-trainers.md (hunk 1 above)
Comment (ChristinaLK) [reply minutes/2019-03-07-trainers.md]: I think Greg wants to not be google-able.

**Comment by ChristinaLK** [comment]
@raynamharris I think I addressed everything. I don't know who can give you write access (probably @karenword ?), but if you have it, it sounds like next time you can just edit the PR directly. does that sound like the best plan?

**Comment by raynamharris** [comment]
Awesome.  Thanks @ChristinaLK

**Comment by karenword** [comment]
I don't seem to have the ability to grant write access, but I have passed this along to those who do. :)

**Pull request merged**